cdap-docs/developers-manual/source/getting-started/standalone/docker.rst

```diff
@@ -49,7 +49,7 @@ Kitematic and then download, start, and connect to an instance of CDAP.
 
 #. Select the desired version.
    Note that the tag **latest** is the last version that was put up at Docker Hub, which is not the 
-   necessarily the most-current version:
+   necessarily the most-current version, which is |literal-version|:
```

Comment:
Let's change "most-current" to "desired" here, since this doc may be referenced after additional releases are made. A user reading the 3.5.8 document should be pointed to the 3.5.8 (or whatever... example) tag of the Docker image from Docker Hub.